.github/workflows/release-homebrew.yaml

```
@@ -14,5 +14,5 @@ jobs:
         tap: microsoft/git
         name: git-credential-manager-core
         type: cask
-        releaseAsset: gcmcore-osx-(.*)\.pkg
+        releaseAsset: gcm-osx-(.*)\.pkg
```

I wonder if we should not also take this opportunity to put the full RID in the asset name? In the future when we have Apple Silicon native packages (`osx-arm64`) we'll need to do this.

```suggestion
        releaseAsset: gcm-osx-x64-(.*)\.pkg
```